shell/edit/workload/index.vue

```diff
@@ -37,15 +37,25 @@ export default {
      */
     mapError(error) {
       switch (true) {
-      case error.includes('violates PodSecurity'): {
```

Comment:
This is meant to be a map, so it have certain cases to be mapped into something else. Although currently we have just one, the new design is aimed to map errors messages into something else and use icons.

We have just to prevent the object case to break and keep the rest of the logic. The `switch` needs to be kept for the substring check, till we have a better logic in place. The way to retrieve the message is the only difference we access to it.

Basically you can bring everything as it was before and just do the following ahead:

```ts
      // If we check for null, we can at this point extend it to undefined, so it does not break in case of undefined
      const isObject = error && typeof error === 'object' && !Array.isArray(error);
      // We have just 2 cases, so we'll set a ternary operation
      const errorMessage = isObject ? error.message || '' : error || '';

      switch (true) {
      case errorMessage.includes('violates PodSecurity'): {
        const match = errorMessage.match(/\"(.*?)\"/gi);
        const name = match[0];
        const policy = match[1];
        return {
          message: `Pod ${ name } Security Policy Violation ${ policy }`,
          icon:    'icon-pod_security'
        };
      }
      default:
        break;
      }
```